config/routes.rb

```diff
@@ -9,14 +9,16 @@
     get "/feedback", to: "feedback#show", as: :feedback_form
     post "/feedback", to: "feedback#submit", as: :feedback_form_submitted
 
-    scope "/login" do
+    scope "/sign-in" do
       get  "/", to: "devise_sessions#create", as: :new_user_session
       post "/", to: "devise_sessions#create"
       get  "/phone/code", to: "devise_sessions#phone_code", as: :user_session_phone_code
       post "/phone/verify", to: "devise_sessions#phone_verify", as: :user_session_phone_verify
       get  "/phone/resend", to: "devise_sessions#phone_resend", as: :user_session_phone_resend
     end
 
+    get "/login", to: redirect("/sign-in")
+
     get "/logout", to: "devise_sessions#destroy", as: :destroy_user_session
```

Comment:
Do we want `/sign-out` to be consistent?

Reply:
I've asked Ale, but this seems like a logical change to me

Reply:
We'll change this too